Cargo.toml

```diff
@@ -40,3 +40,4 @@ zeroize = "1.6.0"
 
 [features]
 fips = ["rusqlite/bundled"]
+slow = [] # Enables slow tests
```

Comment:
should we run the slow tests in CI? Are they enabled by default?

Reply:
They are not enabled by default, and I am not sure we should enable them in CI.
Perhaps we should have a build after a PR merge that makes a pass with slow turned on ...